awx_collection/tests/integration/targets/tower_group/tasks/main.yml

```diff
@@ -11,7 +12,7 @@
 
 - name: Create a Group
   tower_group:
-    name: Some Group
+    name: "{{ group_name }}"
```

Comment:
I would say that for _sub-objects_ this is much less important. They are unique by their own name and their inventory name, so there's no reason you couldn't name them something plain. But the complex name is fine too.